modules/social_features/social_group/modules/social_group_invite/social_group_invite.module

```diff
@@ -706,16 +705,14 @@ function social_group_invite_tokens_alter(array &$replacements, array $context,
       $context["tokens"]["register_link"] === '[group_content:register_link]'
     ) {
       // For new users for register_link token.
-      $link = $replacements["[group_content:register_link]"];
-      $replacements["[group_content:register_link]"] = "{$link}&{$destination}";
+      $replacements["[group_content:register_link]"] = $group_invite_url;
```

Comment:
This change is causing regression.

As admin if you send email invitation to user without account it is expected that invitation link includes user registration.

For example: 

If flexible group is visible to community or public the link should be something like:
https://example.com/user/register?invitee_mail={invite_token}&destination=/social-group-invite/{invite_id}/accept but with given fix it is https://example.com/social-group-invite/{invite_id}/accept and this is wrong

If flexible group is visible to members only the link should be something like:
https://example.com/user/register?invitee_mail={invite_token}&destination=/my-invites but with given fix it is https://example.com/my-invites and this is wrong

Reply:
That's a great find, thanks! I updated the code, so now it uses the destination in the register link only.

Reply:
Tnx for update Nick, links have now the expected pattern.